src/keychain/index.js

```diff
@@ -503,6 +503,36 @@ class Keychain {
       return throwDelayed(errcode(new Error(`Key '${name}' does not exist. ${err.message}`), 'ERR_KEY_NOT_FOUND'))
     }
   }
+
+  async changeKeychainPassword(oldPassword, newPassword){
```

Comment:
```suggestion
  async rotateKeychainPass (oldPass, newPass){
```

Perhaps rotate is a better naming, and we use pass everywhere, so I think this should also use `pass`

Reply:
We probably need to specify the key we want to rotate?